go/roothash/api/commitment/executor.go

```diff
@@ -32,6 +32,7 @@ var (
 //
 // Keep the roothash RAK validation in sync with changes to this structure.
 type ComputeResultsHeader struct {
+	Round        uint64           `json:"round"`
```

Comment:
Yeah, it is just to make it easier to determine if the commit is old. Also it will be useful when we decide to slash -- signing two different commits for the same round should be slashable.

Reply:
ok